src/components/__test__/display.test.js

```diff
@@ -16,7 +16,9 @@ describe('Test Display Component', () => {
   });
 
   test('Testing Diplay snapShot ', () => {
-    const tree = renderer.create(<display valueTotal={6} />).toJSON();
```

Comment:
**Importance of UI testing**

I personally do not have experience with `@testing-library/react`, only with `react-test-renderer` and [Cypress](https://www.cypress.io). However, I would like to still highlight one topic about UI tests.

These tests are usually time expensive and hard to maintain in comparison with unit tests. So it is important to put some thought into what kind of strategy you want to have about UI testing. We can eventually discuss this bit more in a detail on a call.

Reply:
Alright, i look forward to discuss more on this topic